surefire-provider/src/main/java/org/arquillian/smart/testing/surefire/provider/InfoPrinter.java

```diff
@@ -0,0 +1,62 @@
+package org.arquillian.smart.testing.surefire.provider;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+import java.util.stream.IntStream;
+
+import static java.lang.String.format;
+
+public class InfoPrinter {
+
+    private final Logger log = Logger.getLogger(InfoPrinter.class.getName());
+
+    int length;
+    private List<String> strategies;
+    private boolean isSelecting;
+    private String title = "Smart testing";
+    private String surroundingCharacter = "=";
```

Comment:
Why `title` and `surroundingCharacter` are not constants if we don't have any way of changing those?